json-server.py

```diff
@@ -89,6 +105,15 @@ def do_POST(self):
 # If true/false... More detailed responses for failing to create user
 
 
+
+
+
+        
+
+
+
+
+
```

Comment:
Please remove all of this unnecessary white space, please

Reply:
dito ^

Reply:
Fixed and re-pushed